labelbox/orm/comparison.py

```diff
@@ -60,7 +58,8 @@ def __eq__(self, other):
             (self.first == other.second and self.second == other.first))
 
     def __hash__(self):
-        return hash(self.op) + 2833 * hash(self.first) + 2837 * hash(self.second)
+        return hash(
+            self.op) + 2833 * hash(self.first) + 2837 * hash(self.second)
```

Comment:
style nit: ewwwww can we revert this? or get the linter to ignore this instance?